app/services/auth.js

```diff
@@ -197,8 +201,8 @@ export default Service.extend({
 
   syncingDidChange: observer('isSyncing', 'currentUser', function () {
     const user = this.currentUser;
-    if (user && user.get('isSyncing') && !user.get('syncedAt')) {
-      return this.router.transitionTo('first_sync');
+    if (user && !user.get('syncedAt')) {
+      this.first_sync = true;
```

Comment:
Hmm, I thought that `this.set` was still required for stuff like this, but I guess this seems to work?

Reply:
Yes it works perfectly. Thank you @sorta